datalad/support/annexrepo.py

```diff
@@ -157,26 +189,26 @@ def set_direct_mode(self, enable_direct_mode=True):
         """
         if self.is_crippled_fs() and not enable_direct_mode:
             raise CommandNotAvailableError(cmd="git-annex indirect",
-                                           msg="Can't switch to indirect mode on that filesystem.")
-        mode = 'direct' if enable_direct_mode else 'indirect'
-        self.cmd_call_wrapper.run(['git', 'annex', mode], cwd=self.path,
-                                  expect_stderr=True)
-        # TODO: 1. Where to handle failure?
+                    msg="Can't switch to indirect mode on that filesystem.")
```

Comment:
nitpick/pep8(?):  better then to move ```cmd=``` also on the new line so all args (cmd and msg) are identically indented